Multisig/Features/Connect to Web/Logic/WebConnectionController.swift

```diff
@@ -641,10 +641,17 @@ class WebConnectionController: ServerDelegateV2, RequestHandler, WebConnectionSu
         case let sendTxRequest as WebConnectionSendTransactionRequest:
             guard let from = sendTxRequest.transaction.from,
                   let address = Address(from),
-                  connection.accounts.contains(address) else {
+                  connection.accounts.contains(address)
+            else {
                 try? server.send(Response(request: wcRequest, error: .invalidParams))
                 return
             }
+
+            // reject ledger nano x because it is not yet supported for sending transactions
+            if let key = try? KeyInfo.firstKey(address: address), key.keyType == .ledgerNanoX {
```

Comment:
Should we show a proper message to the user as well? Otherwise it may be confusing